config_template.yml

```
@@ -413,3 +413,110 @@ apis:
       deepseek-reasoner:
         aliases: ["r1"]
         max-input-chars: 384000
+  # GitHub Models
+  # https://github.com/marketplace/models
+  github-models:
+    base-url: https://models.github.ai/inference
+    api-key:
+    api-key-env: GITHUB_PAT
```

```suggestion
    api-key-env: GITHUB_PERSONAL_ACCESS_TOKEN
```

To match GitHub MCP server and other tools